calls.go

```diff
@@ -216,6 +216,35 @@ type CallDTMFResponseBody struct {
 	ApiID   string `json:"api_id,omitempty" url:"api_id,omitempty"`
 }
 
+type CreateMaskingSessionParams struct {
+	FirstParty              string `json:"first_party,omitempty" url:"first_party,omitempty"`
+	SecondParty             string `json:"second_party,omitempty" url:"second_party,omitempty"`
+	CallMe                  bool   `json:"call_me,omitempty" url:"call_me,omitempty"`
+	PowerPackUUID           string `json:"powerpack_uuid,omitempty" url:"powerpack_uuid,omitempty"`
+	CallbackUrl             string `json:"callback_url,omitempty" url:"callback_url,omitempty"`
+	SessionExpiry           int    `json:"session_expiry" url:"session_expiry,omitempty"`
+	CallTimeLimit           int    `json:"call_time_limit,omitempty" url:"call_time_limit,omitempty"`
+	RingTimeout             int    `json:"ring_timeout,omitempty" url:"ring_timeout,omitempty"`
+	FirstPlayUrl            string `json:"first_play_url,omitempty" url:"first_play_url,omitempty"`
+	SecondPlayUrl           string `json:"second_play_url,omitempty" url:"second_play_url,omitempty"`
+	Record                  bool   `json:"record,omitempty" url:"record,omitempty"`
+	RecordFileFormat        string `json:"record_file_format,omitempty" url:"record_file_format,omitempty"`
+	RecordingCallbackUrl    string `json:"recording_callback_url,omitempty" url:"recording_callback_url,omitempty"`
+	RecordingCallbackMethod string `json:"recording_callback_method,omitempty" url:"recording_callback_method,omitempty"`
+}
+
+type CreateMaskingSessionResponse struct {
```

Comment:
in case of error one field error will also be returned

Reply:
The "error" field in case of non-2XX response is internally handled and is not required to add as a field in the response struct. In case of an error, SDK will return this response:
Error{"api_id": "97bea9dd-8e0f-48b0-82c9-e3501c2ad5bb","error":"mandatory parameters missing. second_party must be present"}